monkey/agent_plugins/exploiters/log4shell/src/log4shell_exploiter.py

```diff
@@ -214,7 +213,7 @@ def _exploit_ports(
                 options=options,
                 ldap_port=self._ldap_port,
                 agent_binary_downloaded=agent_binary_downloaded,
-                exploit_class_downloaded=self._exploit_class_http_server.bytes_downloaded,
+                exploit_class_downloaded=self._exploit_class_http_server.bytes_downloaded,  # type: ignore [union-attr]  # noqa: E501
```

Comment:
I'm not getting a mypy error here. Can you share what the error is? Is there a better solution than ignoring it?

Reply:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

monkey\agent_plugins\exploiters\log4shell\src\log4shell_exploiter.py:216: error: Item "None" of "Optional[Any]" has no attribute "bytes_downloaded"  [union-attr]
Found 1 error in 1 file (checked 1 source file)
```